deploy/lvm-operator.yaml

```
@@ -626,6 +626,9 @@ rules:
   - apiGroups: [""]
     resources: ["nodes"]
     verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["get", "list", "watch", "update", "patch"]
   - apiGroups: ["*"]
```

Specify the API group here.
```suggestion
  - apiGroups: ["local.openebs.io"]
```